static/bidder-info/loyal.yaml

```diff
@@ -0,0 +1,14 @@
+endpoint: "https://us-east-1.loyal.app/pserver"
```

Comment:
Do you have endpoints in other regions? Is only US traffic supported?

Reply:
We do not have endpoints in other regions and only support US traffic

Reply:
Gotcha. Please add the following in this yaml file to declare US traffic only:

```
geoscope:
  - USA
```

See https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#bidder-info for further info on the geoscope.

Reply:
done